src/crimson/os/cyan_store.h

```diff
@@ -37,6 +37,11 @@ class CyanStore {
 				   uint64_t offset,
 				   size_t len,
 				   uint32_t op_flags = 0);
+  seastar::future<ceph::bufferptr> get_attr(CollectionRef c,
+					    const ghobject_t& oid,
+					    string_view name);
```

Comment:
May we have the full namespace for  the `string_view`?

Reply:
fixed and repushed.